### ISSUE TITLE
Lambda environment variables get CamelCased

### ISSUE THREAD

**Issue by l0st3d** [brabster/crucible#101]
It looks like the configs get camel cased by default. This looks like a source of bugs.

**Comment by brabster** [comment]
Yeah I can see how that would be confusing... not sure exactly what to do as changing the default behaviour would break existing code
